web_development_101/the_front_end/project_html_css.md

```diff
@@ -1864,6 +1864,7 @@ Submit a pull request with a link to your solution in this [file](https://github
 * [MarcusCole518's Solution (Easy)](https://github.com/MarcusCole518/Google-Homepage)
 * [Nathaniel Hill's Solution (Easy)](https://github.com/NHill92/google-homepage) - [View in Browser (Easy)](https://nhill92.github.io/google-homepage/)
 * [Moses Ogwo's Solution (Easy)](https://github.com/mosesogwo/google-homepage) - [View in Browser (Easy)](https://mosesogwo.github.io/google-homepage/)
+* [Chase Wickett's Solution (Easy)](https://wickettc.github.io/google-homepage/) -[View in Browswer (Easy)](https://wickettc.github.io/google-homepage/)
```

Comment:
Both links are the same here, the first one should show your github repository so others can look at your code :)

Also `-[View in Browswer (Easy)](` should be `- [View in Browswer (Easy)](` (Notice the space after the `-`?